driving_gridworld/obstacles.py

```diff
@@ -49,3 +51,22 @@ def expected_reward_for_collision(self, speed):
 
     def __str__(self):
         return 'p'
+
+
+class CarObstacle(Obstacle):
+    def __init__(self, row, col, prob_of_appearing=0.2, obst_speed=1):
+        self.row = row
+        self.col = col
+        self.prob_of_appearing = prob_of_appearing
+        self.obst_speed = obst_speed
+
+    def expected_reward_for_collision(self, speed):
+        return -8e2**(speed + self.obst_speed)
+
+    def reward_for_collision(self, speed, stddev=0.0):
```

Comment:
Since all obstacles can move, `Obstacle#reward_for_collision` should be modified like this, and `CarObstacle` shouldn't need to override it.